python/plugins/processing/algs/gdal/rasterize.py

```diff
@@ -237,6 +235,12 @@ def getConsoleCommands(self, parameters, context, feedback, executing=True):
         arguments.append('-of')
         arguments.append(output_format)
 
+        if input_details.open_options:
```

Comment:
Support for open options was added in GDAL 3.7, maybe add a check for GDAL version here?